### PR TITLE
Brackets memory db fix update by id

### PR DESCRIPTION
**Pull request by ghost**
This adresses the following issue: https://github.com/Drarig29/brackets-storage/issues/12

* fix: add missing rfdc package

* fix: add missing ts-ignore

* fix: access entities on update by found index

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "author": "Corentin Girard",
     "license": "ISC",
     "dependencies": {
-        "brackets-manager": "^1.3.9"
+        "brackets-manager": "^1.3.9",
+        "rfdc": "^1.4.1"
```
Comment (Tandashi) [inline package.json:13]: Why has this dependency been added?

Comment (ghost) [reply package.json:13]: the package is imported in brackets-memory-db/index.ts the original file I edited and has been used there before my change. Therefore I decided to add it.

Comment (Drarig29) [reply package.json:13]: Thanks for adding it, this must have been an oversight

**Review by Tandashi — APPROVED** [review]
Looks good to me.

@Drarig29 Have also verified that the issue is now fixed using the following:

```typescript
import { InMemoryDatabase } from "./brackets-storage/brackets-memory-db";
const storage = new InMemoryDatabase();

async function main() {
  const id = await storage.insert("stage", { first: true });
  const id2 = await storage.insert("stage", { second: true });

  console.log(await storage.select("stage")); // [ { id: 0, first: true }, { id: 1, second: true } ]

  await storage.delete("stage", { id });
  await storage.update("stage", id2, { second: true, updated: 1 });

  console.log(await storage.select("stage")); // [ { second: true, updated: 1 } ]
}
main();
```

Also have added this in the issue as a replication example.

**Review by Drarig29 — APPROVED** [review]

**Pull request merged**